quirks/shared-credentials.json

```diff
@@ -135,6 +135,22 @@
             "diversalertnetwork.org"
         ]
     },
+    {
+        "shared": [
+            "digi.ro",
+            "digionline.ro"
```

Comment:
Maybe the expedient thing to do here is move “digionline.ro” into the “to” of the other rule, even if it’s not 100% accurate. What do you think?

Reply:
Makes sense

Reply:
Actually, `digionline.ro` is NOT obsoleted, so I'm not sure what implications does this change have.

Would
```json
{
    "from": [
        "digionline.ro",
    ],
    "to": [
        "digi.ro"
    ],
},
{
    "from": [
        "digiromania.ro",
        "rcs-rds.ro"
    ],
    "to": [
        "digi.ro"
    ],
    "fromDomainsAreObsoleted": true
}
```
work?

Reply:
Nope, it wouldn't work:
```sh
$ ruby .github/workflows/lint-scripts/websites-shared-credentials-duplicates.rb
The domain 'digi.ro' appears more than once!
```
:(